upstox_client/feeder/portfolio_data_feeder.py

```diff
@@ -37,3 +41,24 @@ def connect(self):
 
         threading.Thread(target=self.ws.run_forever,
                          kwargs={"sslopt": sslopt}).start()
+
```

Comment:
More concise version of this function could be

`  def get_websocket_url(self):
      ws_url = "wss://api.upstox.com/v2/feed/portfolio-stream-feed"
      update_types = []
      if self.order_update:
          update_types.append("order")
      if self.holding_update:
          update_types.append("holding")
      if self.position_update:
          update_types.append("position")
  
      if update_types:
          ws_url += "?update_types=" + "%2C".join(update_types)
  
      return ws_url`

please test and use.